src/UiTPAS/Event/CommandHandling/Validation/EventHasTicketSalesCommandValidator.php

```diff
@@ -43,6 +49,13 @@ public function validate($command)
 
         $eventId = $command->getItemId();
 
+        try {
+            $this->placeRepository->load($eventId);
```

Comment:
This is now a double negotiation, so maybe it's better to inject the event repository?
When the event is not found a return can be done.